src/tools/pontoon/adding_new_locale.md

```diff
@@ -21,6 +21,10 @@ It’s the locale code, in this case `am`.
 
 Google Translate maintains a list of supported locales in its own format. Choose one that matches the locale from [a list of supported locales](https://translate.google.com/intl/en/about/languages/) or leave it blank to disable support for Google Translate for this locale.
 
+### Google AutoML model
+
+Set your [Google Cloud AutoML Translation](https://cloud.google.com/translate/) model ID to use translation engine with a custom model. If the value is not set, Google’s generic machine translation technology is used instead.
```

Comment:
```suggestion
Set your [Google Cloud AutoML Translation](https://cloud.google.com/translate/) model ID to use the translation engine with a custom model. If the value is not set, Google’s generic machine translation technology is used instead.
```

Would it be worth explaining where that ID is coming from? I was expecting something completely different before I looked at Slovenian.